python/cudf/cudf/core/frame.py

```diff
@@ -1918,6 +1873,17 @@ def __copy__(self):
     @_cudf_nvtx_annotate
     def __invert__(self):
         """Bitwise invert (~) for integral dtypes, logical NOT for bools."""
+
```

Comment:
Can we define `_apply_inverse_column` outside of this `__invert__` separately? That way we will not keep re-defining this method when `__invert__` is invoked.

Reply:
You gave me the idea to define `__invert__` on the `Column` so we can avoid this helper all together